wgsl/index.bs

```diff
@@ -8722,9 +8725,8 @@ Together with the shader's stage, these identify the binding address
 of the resource on the shader's pipeline.
 See [[WebGPU#pipeline-layout|WebGPU &sect; GPUPipelineLayout]].
 
-Bindings [=shader-creation error|must not=] alias within a [=shader=]:
-two different variables in the resource interface of a given
-shader [=shader-creation error|must not=] have the same group and binding values, when considered as a pair of values.
+It is a [=dynamic error=] if two mutable resource variables have the same pair
```

Comment:
This is not the kind of aliasing that is wanted.

They want the same memory locations to be accessilble from resources with distinct (group,binding).
Not that they want to allow two variables with the same (group,binding). 

Maybe it's best to avoid using the word "aliasing" here at all, and simplify:

> Two different resource variables in a [=shader=] [=...|must not=] have the same group and binding values, when considered as a pair.